utils.sh

```diff
@@ -56,6 +63,58 @@ function patch_node_ignition() {
     ssh < "${wd}/${kind}.yaml" "${ssh_opts[@]}" "core@$bootstrap" sudo dd of="$config"
 }
 
+function machineconfig_generate_patches() {
+    set -x
+    local kind
+    local path
+    local value
+    local search_path
+    local config_type
+    local files_template
+    local units_template
+    local octal_mode
+    local decimal_mode
+    local dest_path
+    local ifs
+    local simple_name
+
+    read -r -d '' files_template <<'EOF'
```

Comment:
bashism, this won't work when `sh ./ocp_run` is called

Reply:
the file is headed with #!/bin/bash are we really not able to depend on bash?

Reply:
If that is the case, we lose `local` echo flags, etc

Reply:
hmm, weird, it seems its running as a bash but it still fails here